### PR TITLE
app-emulation/sen: Initial commit

### PR DESCRIPTION
**Pull request by ntnn**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

Ping at my mentor: @Polynomial-C

### CODE REVIEW

File: app-emulation/sen/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		<email>nelo@wallus.de</email>
+		<name>Nelo-Thara Wallus</name>
+		<description>Proxied by Polynomial-C</description>
+	</maintainer>
+	<maintainer type="person">
+		<email>polynomial-c@gentoo.org</email>
+		<name>Lars Wendler</name>
+		<description>Proxy maintainer for Nelo</description>
+	</maintainer>
+	<upstream>
```
Comment (mgorny) [inline app-emulation/sen/metadata.xml]: Don't you want proxy-maint to help here?

File: app-emulation/sen/sen-0.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
```
Comment (mgorny) [inline app-emulation/sen/sen-0.5.1.ebuild]: Could you test 3.6 as well? If you plan to maintain multiple Python packages, that would be really helpful.

File: app-emulation/sen/sen-0.5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+DESCRIPTION="Terminal User Interface for docker engine"
+HOMEPAGE="https://github.com/TomasTomecek/sen"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="dev-python/urwid
```
Comment (mgorny) [inline app-emulation/sen/sen-0.5.1.ebuild]: Dependencies on Python modules always require `[${PYTHON_USEDEP}]` to enforce implementation match.

File: app-emulation/sen/sen-0.5.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	dev-python/urwidtrees
+	dev-python/docker-py"
+RDEPEND="${DEPEND}"
+
+DOCS="docs/*"
```
Comment (mgorny) [inline app-emulation/sen/sen-0.5.1.ebuild]: The non-array variant of `DOCS` is generally discouraged, and so is implicit globbing. Also, this implicitly disables installing `README*` and other defaults. Preferably you'd redefine `python_install_all` and install extra files there.

File: app-emulation/sen/sen-0.5.1.ebuild (hunk 2 above)
Comment (ntnn) [inline app-emulation/sen/sen-0.5.1.ebuild]: 3.6 currently isn't supported by urwid (and urwidtrees), which are dependencies of app-emulation/sen, hence no 3.6. Once urwid has resolved all issues on 3.6 I wanted to make a new patch set enabling python3_6 for sen and the dependencies.

**Pull request closed without merging**